examples/webgl_mirror.html

```diff
@@ -81,8 +83,8 @@
 				geometry = new THREE.CircleGeometry( 40, 64 );
 				groundMirror = new Reflector( geometry, {
 					clipBias: 0.003,
-					textureWidth: window.innerWidth * window.devicePixelRatio,
-					textureHeight: window.innerHeight * window.devicePixelRatio,
+					textureWidth: window.innerWidth * resolution,
+					textureHeight: window.innerHeight * resolution,
```

Comment:
Regarding #31697, I would suggest you add `resolutionScale` to `Reflector` instead.

Ideally, the reflector is also created based on the drawing buffer size so users get by default a perfect mirror. I would like to get rid of DPR handling on app level as much as possible. Setting the DPR in `setPixelRatio()` should be sufficient.

Reply:
I set the scale to 0.5 because the performance seemed jittery when resizing otherwise. I'll change it to 1 if you insist.